docs/components/card.md

```diff
@@ -995,3 +995,46 @@ Card columns can also be extended and customized with some additional code. Show
   }
 }
 {% endhighlight %}
+
+## Publication
+Shows implementations of the publication element.
+
+{% example html %}
+<div class="card">
+  <article class="card-block card-block-action publication-block has-arrow">
+    <header class="publication-header">
+      <a class="publication-title" href="#" title="Personalsachbearbeiter/in in Grünwald" itemprop="title">Lorem ipsum dolor sit amet, consetetur sadipscing elitr</a>
```

Comment:
Leave it in, it should be convention that if we list a publication we include the necessary microformats. Since this component is very specifically about the publication it's good to include it.